src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -117,6 +118,8 @@ def __init__(
                     friendly_coef=1.0,
                     unrealized_coef=1.0,
                     discount_factor=self.discount_factor,
+                    random_betrayal=False,
+                    end_game_flip=False,
```

Comment:
These should be added for the other `ActionBasedStance` constructors as well.

Reply:
sounds good, will do